test/test-group.js

```diff
@@ -302,6 +302,28 @@ describe('Groups', function () {
       });
     });
 
+    // it('should remove last member from group and delete group ', function () {
+    //   User.findById(user2.id)
```

Comment:
commented out for now until I have time to write a better test, but it def works because I've tested it on front end!